test/glint_test.gleam

```diff
@@ -10,10 +10,7 @@ pub fn main() {
 
 pub fn path_clean_test() {
   glint.new()
-  |> glint.add(
-    ["", " ", " cmd", "subcmd\t"],
-    glint.command(fn(_, _, _) { Nil }),
-  )
+  |> glint.add(["", " ", " cmd", "subcmd"], glint.command(fn(_, _, _) { Nil }))
```

Comment:
⚠️ this tab character after `subcmd` was there to test the whitespace stripping of `glint.add` and shouldnt be changed as part of this

Reply:
Reverted